Bottom: 5f171fa19953f9350c60baf9a3a30bf33c41bbbe
Top:    48e88aa5359fa15b806b6835dea279daed69cc2e
Author: Ian Kent <raven@themaw.net>
Date:   2013-03-06 11:17:19 +0800

Refresh of autofs-5.0.7-dont-fail-on-master-map-self-include.patch

---

diff --git a/modules/lookup_file.c b/modules/lookup_file.c
index 0d378ad..f37bed9 100644
--- a/modules/lookup_file.c
+++ b/modules/lookup_file.c
@@ -399,7 +399,7 @@ int lookup_read_master(struct master *master, time_t age, void *context)
 
 	/* Don't return fail on self include, skip source */
 	if (master->recurse)
-		return NSS_STATUS_SUCCESS;
+		return NSS_STATUS_TRYAGAIN;
 
 	if (master->depth > MAX_INCLUDE_DEPTH) {
 		error(logopt, MODPREFIX
